BP/common/culture/cultures/00_continental_celtic.txt

```diff
@@ -423,50 +485,27 @@
 		color = { 0.3 0.8 0.8 }
 
 		cadet_dynasty_names = {
-			"dynn_Valetilus"
+				"dynn_Valetilus"
+				"dynn_Antorbanen"
+				"dynn_Sekaiza"
+				"dynn_Deobrigo"
+				"dynn_Segisamo"
 		}
 
 		dynasty_names = {
-			"dynn_Valetilus"
+				"dynn_Valetilus"
+				"dynn_Antorbanen"
+				"dynn_Sekaiza"
+				"dynn_Deobrigo"
+				"dynn_Segisamo"
 		}
 
 		male_names = {
 			Antorbanen Bodo Sekaiza Cado Deobrigo Dovidero Durato Elguismio Segisamo Turenno  
-			Aaron AbbA_n Abner AbE_l AdomnA_n A_ed A_edgen A_educA_n A_edA_n A_engus Affiath Ailbe Ailbrend Ailbrenn AilchU_ Aildobur Ailgel A_ilgenA_n Ailgus
```

Comment:
Why all the name removals here?

Reply:
So - it looked as if a big block of specifically Irish Gaelic names had been added here, which looked a bit... off for any Continental Celtic culture. 

I did have a wee look to see if there were ever records of such Irish Gaelic names in northern Iberia, but couldn't find any. I took it to be a copy error of some sort.

That said, v. happy if I'm wrong! I know about Galicia, and the migration there, but that would be post-game start and probably better represented by fleeing Britons establishing a toehold, rather than Irish names within one of the native Celtic cultures.

Reply:
Makes sense, do you think it would be possible to find alternative names for the Cantabrian culture? Do you have any resources in mind or the time to work on this task?